cpp/include/typedb/concept/type/attributetype.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#pragma once
+
+#include "typedb/concept/type/thingtype.hpp"
+
+namespace TypeDB {
+
+class AttributeType : public ThingType {
+    friend class Concept;
+
+   private:
+    AttributeType(_native::Concept* conceptNative);
```

Comment:
Missing symbols have since been implemented. 

Is there a way to check for these missing symbols before they're called? Should I add `-z defs` (or whatever the right one is) to my linker options?